src/cdk-experimental/menu/menu-item.spec.ts

```diff
@@ -3,60 +3,94 @@ import {ComponentFixture, TestBed, async} from '@angular/core/testing';
 import {By} from '@angular/platform-browser';
 import {CdkMenuModule} from './menu-module';
 import {CdkMenuItem} from './menu-item';
+import {CDK_MENU} from './menu-interface';
+import {CdkMenu} from './menu';
 
 describe('MenuItem', () => {
-  let fixture: ComponentFixture<SingleMenuItem>;
-  let button: CdkMenuItem;
-  let nativeButton: HTMLButtonElement;
-
-  beforeEach(async(() => {
-    TestBed.configureTestingModule({
-      imports: [CdkMenuModule],
-      declarations: [SingleMenuItem],
-    }).compileComponents();
-  }));
-
-  beforeEach(() => {
-    fixture = TestBed.createComponent(SingleMenuItem);
-    fixture.detectChanges();
-
-    button = fixture.debugElement.query(By.directive(CdkMenuItem)).injector.get(CdkMenuItem);
-    nativeButton = fixture.debugElement.query(By.directive(CdkMenuItem)).nativeElement;
-  });
+  describe('without enclosing elements', () => {
```

Comment:
I thought this meant, without elements *around* the menu item. Could you reword this to be clearer (eg. 'with no complex inner elements', to match the other suite).